README.md

```diff
@@ -1,6 +1,6 @@
 # Hello-World
 This is simple Hello World Prject
-![screenshot](./hello.png)
+![screenshot](./hello1.png)
```

Comment:
Your Readme looks promising, but you're having some placeholder texts (e.g. `This is an example of how you may give instructions on setting up your project locally`) in the  `Getting Started` section. I think you should replace the placeholder text with what actually resonates with this project. The text is there to guide you on what to do in the section :blush: 

Note that you can remove completely if any of the subsections (e.g. deployment) is not applicable to the project.

Reply:
I try to fix things but when I change the branch name this pull request will be deleted

Reply:
Hello @ridaarif98,

I do not mean to mislead you. I myself did not realize that renaming the branch will close the PR. Thanks for letting me know.

If renaming the branch will close/delete this PR, I suggest you should not bother to rename the branch. You can restore the branch, then go to the PR page for the project and mark the project as open.

**If you've already created and submitted a new PR, you can ignore this message.** However, you should take note of this against your subsequent projects.